string behavior_name
bool active